src/actions/guides/database/pagination.cr

```diff
@@ -0,0 +1,422 @@
+
+    Next, let's do something a bit more realistic. We'll modify the
+    `BaseSerializer` to accept a `Lucky::Paginator` when rendering a
+    collection. That way we can add pagination data to every JSON response that
+    renders a collection.
+
+    ```crystal
+    # src/serializers/base_serializer.cr
+    abstract class BaseSerializer < Lucky::Serializer
+      # Add a new 'pages' argument
+      def self.for_collection(collection : Enumerable, pages : Lucky::Paginator, *args, **named_args)
+        {
+          "items" => collection.map do |object|
+            new(object, *args, **named_args)
+          end,
+          # Add pagination metadata to the response
+          "pagination" => {
+            next_page: pages.path_to_next,
+            previous_page: pages.path_to_previous,
+            total_items: pages.item_count,
+            total_pages: pages.total
+          }
+        }
+      end
+    end
+    ```
+
+    If you want to customize rendering further, see the [collection rendering
+    guide](#{Guides::JsonAndApis::RenderingJson.path(anchor: Guides::JsonAndApis::RenderingJson::ANCHOR_CUSTOMIZING_COLLECTION)}).
+
+    Next, we'll go into details about all the methods available on the
+    `Lucky::Paginator` object.
+
+    ## The Lucky::Paginator object
+
+    The `paginate` method returns a `Lucky::Paginator` object. This object
+    contains information about the pages as well as a number of methods for
+    generating paths to a given page.
+
+    ### `page`
+
+    The page method returns the requested page as an Int32
+
+    ### `item_count`
+
+    The total number of items in the query as an `Int32 | Int64`
+
+    ### `one_page?`
+
+    Returns `true` if there is just 1 page.
+
+    ### `last_page?`
+
+    Returns `true` if you are on the last page.
+
+    ### `first_page?`
+
+    Returns `true` if you are on the first page.
+
+    ### `total`
+
+    Returns the total number of pages as an `Int64`
+
+    ### `overflowed?`
+
+    Returns `true` if the requested page is past the last page.
+
+    ### `path_to_next`
+
+    Returns the path with a 'page' query param for the next page.
+
+    Returns `nil` if there is no next page.
+
+    ### `path_to_previous`
+
+    Returns the path with a 'page' query param for the previous page.
+
+    Returns `nil` if there is no previous page.
+
+    ### `path_to_page`
+
+    Returns the path with a 'page' query param for the given page.
+
+    ### `item_range`
+
+    Returns the `Range` of items on this requested page.
+
+    For example if you have 50 records, showing 20 per page, and you are on
+    the 2nd page this method will return a range of 21-40.
+
+    You can get the beginning and end by calling `begin` or `end` on the
+    returned `Range`.
+
+    ```crystal
+    range = pages.item_range
+    "Showing #\{range.begin}-#\{range.end} of #\{pages.item_count}"
+    ```
+
+    ### `series`
+
+    Requires its own section 😃
+
+    ## The 'series' method
```

Comment:
These quotes mess up the anchor. This is probably a bug in how these are generated. 

https://lucky-websit-pcs-240-pa-ljxzhi.herokuapp.com/guides/database/pagination#the-'series'-method

Reply:
Ah ok. I'll leave this as-is and do a followup PR that slugifies these correctly!